automock/supervisord.conf

```diff
@@ -0,0 +1,16 @@
+[supervisord]
+nodaemon=true
+
+[program:moto]
+command=/home/autopush/pypy/bin/moto_server dynamodb2 -p 5000
```

Comment:
On second thought, since we can just mount the appropriate config files, we don't need the extra arguments.